database/user.go

```diff
@@ -922,11 +922,11 @@ func monthStartWithTime(subscribedUntil time.Time, current time.Time) time.Time
 //
 // Example:
 // In February normalizeDayOfMonth(31) will return 28 or 29.
-func normalizeDayOfMonth(month time.Month, day int, current time.Time) int {
-	t := time.Date(current.Year(), month, day, 0, 0, 0, 0, time.UTC)
+func normalizeDayOfMonth(year int, month time.Month, day int) int {
```

Comment:
Because while `time.Month` is an `int`, `time.Date()` takes a `time.Month` and `time.Time{}.Month()` returns a `time.Month`. So, this approach requires no type casting while the `year, month, day int` approach (while looking nicer as a signature) will require type casting.

Reply:
hm ok sure